cdk-testsuite/utils/openshiftUtils.py

```diff
@@ -0,0 +1,14 @@
+'''
+Created on Jun 29, 2016
+
+@author: amit
+'''
+
+version = '1.0'
+release = ''
+
+def get_version():
```

Comment:
yes it can be. I put it in utils/openshiftUtils.py to make the test suite more modular and any body can access the version without touching the core library.